### PR TITLE
Update conveners for 2021

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
Try to make the links to email the conveners roughly
standard, with one "mailto:" to all (though some groups
maintain separate convener email addresses).

### CODE REVIEW

File: _workinggroups/dataanalysis.md [hunk 1]
```diff
@@ -24,9 +24,9 @@ well-defined set of problems.
...
 [Forum](https://groups.google.com/forum/#!forum/hsf-analysis-wg)
 
-Convenors: Chris Burr (LHCb), Allison Hall (CMS), Paul Laycock (Belle II), TJ Khoo (ATLAS)
+Convenors: Chris Burr (LHCb), Allison Hall (CMS), TJ Khoo (ATLAS)
 
 ---
 
 Former Convenors:
-- Danilo Piparo (ROOT), Andrea Rizzi (CMS)
+- Danilo Piparo (ROOT), Andrea Rizzi (CMS), Paul Laycock (Belle II),
```
Comment (eduardo-rodrigues) [inline _workinggroups/dataanalysis.md:32]:
```suggestion
- Danilo Piparo (ROOT), Andrea Rizzi (CMS), Paul Laycock (Belle II)
```

Comment (eduardo-rodrigues) [reply _workinggroups/dataanalysis.md:32]: Aside question: is that too much of a pain to add time frames to convenerships as we did for PyHEP, see the bottom of https://hepsoftwarefoundation.org/workinggroups/pyhep.html?

File: _workinggroups/detsim.md [hunk 2]
```diff
@@ -19,9 +19,10 @@ technologies) are an important area of work.
 The group's meeting page is [in Indico](https://indico.cern.ch/category/10916/) and
 discussions are held on the [hsf-simulation](https://groups.google.com/forum/#%21forum/hsf-simulation) list.
 
-Convenors: Gloria Corti (LHCb) <mailto:Gloria.Corti@cern.ch>, Philippe Canal (CMS) <mailto:philippe.canal@cern.ch>, Witek Pokorski (Geant4) <mailto:Witold.Pokorski@cern.ch>
+- Convenors: Ben Morgan (ATLAS, Geant4), Kevin Pedro (CMS), Krzysztof Genser (Mu2e+Geant4)
```
Comment (eduardo-rodrigues) [inline _workinggroups/detsim.md:22]:
```suggestion
- Convenors: Ben Morgan (ATLAS, Geant4), Kevin Pedro (CMS), Krzysztof Genser (Mu2e, Geant4)
```

Comment (eduardo-rodrigues) [reply _workinggroups/detsim.md:22]: Suggestion for consistency with Ben's way of providing affiliations.

File: _workinggroups/toolsandpackaging.md [hunk 3]
```diff
@@ -56,15 +56,24 @@ rapidly evaluate each tool for features, ease of use and flexibility.
...
-*  All coordinators can be reached at <hsf-software-tools-wg-conveners@googlegroups.com>
 *  For all technical discussions please use <hsf-tech-forum@googlegroups.com> (Tools) and <hsf-packaging-wg@googlegroups.com> (Packaging)
 
-Former Convenors:
-- Bendikt Hegner (CERN)
-- Elizabeth Sexton-Kennedy (FNAL)
-- Graeme A Stewart (CERN)
+### Former Conveners
+
+- Bendikt Hegner (CMS, EP-SFT)
```
Comment (eduardo-rodrigues) [inline _workinggroups/toolsandpackaging.md:75]:
```suggestion
- Benedikt Hegner (CMS, EP-SFT)
```

File: _workinggroups/toolsandpackaging.md [hunk 4]
```diff
@@ -56,15 +56,24 @@ rapidly evaluate each tool for features, ease of use and flexibility.
...
+- Elizabeth Sexton-Kennedy (CMS)
+- Graeme A Stewart (ATLAS)
+- Ben Morgan (ATLAS and superNEMO)
```
Comment (eduardo-rodrigues) [inline _workinggroups/toolsandpackaging.md:78]: Ben is mentioned as ATLAS and Geant4 above. @drbenmorgan, what  exactly are your affiliations?

**Review by eduardo-rodrigues — APPROVED** [review]
I made some tiny suggestions. Let me pre-approve anyway.

**Comment by graeme-a-stewart** [comment]
> I made some tiny suggestions. Let me pre-approve anyway.

Thanks @eduardo-rodrigues, fixed the small points you found and implemented "terms of office" for all of the previous conveners.

**Review by hegner — APPROVED** [review]

**Review by hegner — APPROVED** [review]

**Review by klieret — APPROVED** [review]
LGTM!

**Pull request merged**